app/templates/components/supplementary-file-browser.hbs

```diff
@@ -5,9 +5,11 @@
 {{#if supplementList}}
     <h5>{{selectedFile.name}}</h5>
     <section>
-        <button class="show-more-button text-muted left" {{action 'moveLeft'}}>
-            <i class="fa fa-caret-left"></i>
-        </button>
+        {{#if showArrows}}
+            <button class="show-more-button text-muted left" {{action 'moveLeft'}}>
```

Comment:
Let's not use text-muted to change the style. It should be applied to items that need to be muted because of being disabled or something similar.